### ISSUE TITLE
restricting transformable area

### ISSUE THREAD

**Issue by padoauk** [skmail/vue-free-transform#6]
Hi.

Vue-free-transform is great. Translatable area restriction capability make it better, I guess. In many cases, area restriction is useful. Without it, translation is so free that object can be released in a area where horizontal position is negative and selection is impossible. 

I found that the restriction is possible by checking payload in update(). But it looks much better to have certain component's property for restriction.

**Comment by devonik** [comment]
Yes that would be nice. I also added a conditional to the update() method but it would be nice to prevent this. Also maybe better peformance
